tests/chainerx_tests/unit_tests/routines_tests/test_normalization.py

```diff
@@ -119,7 +119,7 @@ def setup(self, float_dtype):
         else:
             self.check_forward_options.update({'rtol': 1e-6, 'atol': 1e-5})
             self.check_backward_options.update({
-                'eps': 1e-3, 'rtol': 1e-3, 'atol': 1e-4})
+                'eps': 1e-3, 'rtol': 5e-2, 'atol': 5e-3})
```

Comment:
They're 50 times larger. I guess that's not what's intended.

Reply:
Sorry, let me fix it.

Reply:
Fixed. Made if 5 times larger.